pkgs/tools/networking/aws-xray-daemon/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "aws-xray-daemon-${version}";
+  version = "V3.0.0";
```

It's not typical to have the prefix of V in the version, even if it is in the tag or release name.
```suggestion
  version = "3.0.0";
```